### PR TITLE
Adds modified clearing of inventory

### PR DESCRIPTION
**Pull request by tylerhyperHD**
This command removes stress on admins and clears the inventory of only yourself, meaning people can’t do /clear ** or do /clear (another player). It also overrides essentials by itself, so no configs to worry about on this one.

### CODE REVIEW

**Comment by Wild1145** [comment]
Looks good at first glance however as per the contribution guidelines you do need to either reference this to an issue or open one for discussion.

**Comment by Wild1145** [comment]
Linked to #392

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_clear.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import me.StevenLawson.TotalFreedomMod.TotalFreedomMod;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.ALL, source = SourceType.BOTH)
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_clear.java]: If this is a player-only command, you should set  source = SourceType.PLAYER

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_clear.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+@CommandParameters(
+        description = "Clears your inventory. Made to remove stress on admins.",
+        aliases = "ci",
+        usage = "/<command>")
+public class Command_clear extends TFM_Command
+{
+    @Override
+    public boolean run(final CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+            Player player = (Player)sender;
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_clear.java]: `sender_p`  should be used instead.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_clear.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            player.closeInventory();
+            player.getInventory().clear();
+            player.sendMessage(ChatColor.GOLD + "Your inventory has been cleared.");
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_clear.java]: `playerMsg()` should be used instead.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_clear.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import me.StevenLawson.TotalFreedomMod.TotalFreedomMod;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.ALL, source = SourceType.BOTH)
+@CommandParameters(
+        description = "Clears your inventory. Made to remove stress on admins.",
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_clear.java]: The reason _why_ this command was implemented is irrelevant, please remove it.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_clear.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        aliases = "ci",
+        usage = "/<command>")
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_clear.java]: I'd like to see functionality for mass-clearing player inventories implemented.

**Comment by JeromSar** [comment]
Your commit does not include the required "Closes #392" in your commit message. Please add a commit message including so.

**Comment by tylerhyperHD** [comment]
I had to re-fork the mod, so here is the resolve:
https://github.com/tylerhyperHD/TotalFreedomMod/commit/41377a236b299274776365a6866b80a21c8c4d64

**Comment by Wild1145** [comment]
@tylerhyperHD None of your commits still say "Resolves #392". Until thats added I dont think the PR will be merged.

**Comment by Wild1145** [comment]
@JeromSar There appears to be "No files modified" anymore, so I think this is now a blank PR...

**Comment by JeromSar** [comment]
@tylerhyperHD Please make your changes on this PR.

**Comment by JeromSar** [comment]
@tylerhyperHD Closed per inactivity.

**Pull request closed without merging**